### PR TITLE
Fix: Fixed poor hover effect on the globe icon in the top navigation bar -ZUR-126

### PR DESCRIPTION
**Pull request by Favourz1**
**### This PR fixed the odd grey background color effect when the globe icon on the navigation bar is hovered with issue code  [ZUR-126]**

Before:
zuri_chat_home-before

After:
zuri_chat_home-after

Thanks!

### CODE REVIEW

File: packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css [hunk 1]
```diff
@@ -83,9 +83,6 @@
   height: 25px;
   border-radius: 50%;
 }
-.world:hover {
```
Comment (JulianaSau) [inline packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: You've just done away with the code. So how will a user know if they've hovered on the icon?
I suggest you add a better hover effect, not just remove the code entirely.

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css (hunk 1 above)
Comment (JulianaSau) [inline packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: You've just done away with the code. So how will a user know if they've hovered on the icon?
I suggest you add a better hover effect, not just remove the code entirely.

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css (hunk 1 above)
Comment (Favourz1) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: Ok @JulianaSau  What of if i add a scale effect, So the icon grows bigger in a nice way when hovered with a good transition?

Comment (JulianaSau) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: Yes. That would work.
You could also make the opacity of the color of the icon change when it's hovered on

Comment (JulianaSau) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: You'll tag me when you're done to review your changes again

Comment (Favourz1) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: Ok Making changes now

Comment (Favourz1) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: @JulianaSau  I'm done should i create a new pull request for it?

Comment (JulianaSau) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: Hi @Favourz1, Send a video of how it behaves now

Comment (Favourz1) [reply packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css:86]: @JulianaSau  Please find attached link to the video and how the icon works now.

https://drive.google.com/file/d/1AdGbkH38nzS6WlG1yTgm0V6w-TdMXwml/view?usp=drivesdk

**Review by JulianaSau — APPROVED** [review]

**Comment by Favourz1** [comment]
@JulianaSau   it's yet to be merged.

**Comment by JulianaSau** [comment]
If it has the accepted label it means it will be merged @Favourz1. Just chill

**Comment by Favourz1** [comment]
@ickynavigator  merge Please 🥺.

So I can be graded. Thanks 😊

**Comment by JulianaSau** [comment]
Pull changes from the dev branch to get updates first

**Comment by JulianaSau** [comment]
Your branch is 216 commits behind of the dev branch

**Comment by Favourz1** [comment]
@JulianaSau  Pulled all changes.

**Comment by Favourz1** [comment]
Thank You @JulianaSau !

**Pull request merged**